src/components/common/DatePicker.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
Have you thought about making this a shared component?

Reply:
That was the intention :) 

After share structure, i realized making things shared is about 2Xeasier AFTER meeting the initial design requirements. 

Adjustments will always happen, but things seem to evolve on that end and it becomes tedious hopping back and forth between repo's i'm sure you know :)